config.json

```diff
@@ -1704,6 +1704,21 @@
         "prerequisites": ["arrays", "classes", "array-loops", "objects"],
         "difficulty": 6,
         "topics": ["algorithms", "logic"]
+      },
+      {
+        "slug": "go-counting",
+        "name": "Go Counting",
+        "uuid": "177fc434-027d-48a4-8525-fa3d1d15607e",
+        "practices": [],
+        "prerequisites": [
+          "arrays",
+          "classes",
+          "array-loops",
+          "objects",
+          "conditionals"
```

Comment:
`conditionals-switch` might be a good addition, also I feel this is more focused on `for-loops` than `array-loops`